src/components/Dashboard/MonthlySpend.tsx

```diff
@@ -0,0 +1,61 @@
+//Import CSS file for Monthly Spending styles
+import './MonthlySpending.css';
+//Interface categorizing data by its type
+interface SpendingData {
+    category: string;
+    amount: number;
+    total: number;
+    trend: 'up' | 'down';
+}
+//Function to format numbers with commas for thousands
+function formatNumber(num: number): string {
+    return num.toLocaleString();
+}
+//Functional Component to display monthly spending data
+const MonthlySpending = () => {
```

Comment:
Add dynamic API calling to this. We can make use of Mockaroo to do said such feature. Besides that good work on this.